c++/src/ByteRLE.cc

```diff
@@ -433,6 +441,13 @@ namespace orc {
 
   void ByteRleDecoderImpl::next(char* data, uint64_t numValues,
                                 char* notNull) {
+    AutoStopwatch meassure(&metrics.ByteDecodingLatencyUs,
```

Comment:
This is not the pure decoding time since it contains reading buffers from the input stream. So it's actually ioTime + decompressionTime + decodingTime. I think we should either fix it or document it.